### PR TITLE
KYV-474 Add product experience openapi definition and swagger-ui

### PR DESCRIPTION
**Pull request by synthko**

### CODE REVIEW

**Review by severikupari1 — APPROVED** [review]

**Comment by severikupari1** [comment]
Good work!

File: packages/product-backend/src/service.ts [hunk 1]
```diff
@@ -45,6 +46,14 @@ export const createProductAccounting = async (p: {
   if (!process.env.PRODUCT_BACKEND_URL) {
     throw new Error('No product backend URL set')
   }
+  // These values are required by SAP and need be provided
+  if (
```
Comment (severikupari1) [inline packages/product-backend/src/service.ts:50]: These should be added to yup schema, no need to do custom validations

**Review by severikupari1 — CHANGES_REQUESTED** [review]

File: packages/product-backend/src/service.ts (hunk 1 above)
Comment (severikupari1) [reply packages/product-backend/src/service.ts:50]: These validations are done by controller, so it is easy to add to there.

File: packages/product-experience-api/package.json [hunk 2]
```diff
@@ -6,11 +6,12 @@
   "scripts": {
     "clean": "rimraf dist && rimraf tsconfig.tsbuildinfo",
     "prepack": "yarn build",
-    "build": "yarn clean && yarn compile",
+    "build": "yarn clean && yarn compile && yarn copy:openapi",
     "compile": "tsc --build && cp \"./package.json\" ./dist/",
     "lint": "eslint \"./src/**/*.{ts,tsx}\" --max-warnings=0",
     "start": "node dist/index.js",
-    "dev": "NODE_ENV=development nodemon src/index.ts"
+    "dev": "NODE_ENV=development nodemon src/index.ts",
```
Comment (severikupari1) [inline packages/product-experience-api/package.json:13]: FYI: If you want you could add -L to make product experience api to "hotreload" after file changes.

"dev": "NODE_ENV=development nodemon -L src/index.ts",

File: packages/product-backend/src/service.ts (hunk 1 above)
Comment (synthko) [reply packages/product-backend/src/service.ts:50]: The more you know - this is cleaner and much simpler, thanks!

**Review by severikupari1 — APPROVED** [review]

**Review by martinldev — APPROVED** [review]

**Pull request merged**